### ISSUE TITLE
Training SceneCollisionNet with robot meshes

### ISSUE THREAD

**Issue by adithyamurali** [NVlabs/SceneCollisionNet#2]
Hi Mike, thanks for the great repo.

I was trying to retrain SceneCollisionNet and was wondering if it was trained on robot meshes?  From the paper, it seems the franka link meshes were used (points sampled on the mesh surface) along with the partial object point clouds of ACRONYM objects.

In the code however, in the `get_obj` function, I was only able to see the object meshes: https://github.com/NVlabs/SceneCollisionNet/blob/main/scenecollisionnet/datasets.py#L115

Thanks!

**Comment by mjd3** [comment]
Hi Adithya,

Yeah we did some training with robot meshes, but I removed fully-observed sampling for the code release to avoid confusion. However, if you'd like to add it back in, I'd suggest replacing lines 141-142 [here](https://github.com/NVlabs/SceneCollisionNet/blob/f27cc6305e1a9af5b9878526a13c4fa31b1a2b7e/scenecollisionnet/datasets.py#L141) with:

```
if "franka" in obj_mesh.metadata["key"]:
    points = obj_mesh.sample(self.n_obj_points)
else:
    obj_scene.add_object("obj", obj_mesh, pose)
    points = obj_scene.render_points()
```

Another option (other than looking at specific key names) would be to mark in the mesh metadata whether you want them to be fully-observed during training (i.e., set a metadata entry [here](https://github.com/NVlabs/SceneCollisionNet/blob/f27cc6305e1a9af5b9878526a13c4fa31b1a2b7e/scenecollisionnet/scene.py#L109)) by category and then checking that in the code snippet above. But in terms of the actual mesh sampling, this is already built into trimesh as shown above. Let me know if you run into any issues with it!
